src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -897,12 +896,13 @@ static bool updateAttribute
 * original value of the attribute was different than the one used in the update (this is
 * important for ONCHANGE notifications). Otherwise it is false
 *
-* (Previous versions of this function used the return value for that, but we have
-* modified to make in similar to updateAttribute()
+* In addition, return value is as follows:
+* - true: there was an actual append change
+* - false: there was an append-as-update change
 *
```

Comment:
Perhaps true and false aren't the best return values.
Why not create an enum for it?

E.g.

    type enum MongoUpdateStatus
    {
      MusNormalAppendChange,
      MusAppendAsUpdateChange
    } MongoUpdateStatus;

    static MongoUpdateStatus appendAttribute(...)